web/pages/category/index.tsx

```diff
@@ -0,0 +1,62 @@
+import { Stack, Text, Wrap, Box, Container, Flex } from "@chakra-ui/react";
+import { GetStaticProps } from "next";
+import { TextLink } from "../../features/design-system/TextLink";
+import { getClient } from "../../utils/sanity/sanity.server";
+import { getRainbowColor } from "../../utils/color";
+import { BackButton } from "../../features/post-list/BackButton";
+
+type Category = { name: string; slug: string };
+
+export default function CategoryOverviewPage({
+  categories,
+}: {
+  categories: Category[];
+}) {
+  return (
+    <Box>
+      <Flex
+        flexDirection="column"
+        minHeight="100vh"
+        background="#000"
+        padding="2.5rem"
+      >
+        <Container maxWidth="container.lg">
+          <BackButton color="white" />
+          <Stack direction={"row"} mt={4}>
+            <Wrap>
+              {categories.map((category, index) => (
+                <Text
+                  key={category.name}
+                  fontSize={51}
+                  fontFamily="DINOT"
+                  lineHeight={1.5}
+                  color={getRainbowColor(categories.length, index + 1)}
+                  pr={4}
+                >
```

Comment:
kanskje en hover-effekt hadde vært kult å legge til? Enten å skifte fargen litt, eller gjøre noe litt mer spennende om du vil?

Also, kanskje legge til det samme for focusVisible - dvs når man bruker tastaturet for å tabbe seg rundt.

Reply:
Høres bra ut 💡 😃 
Will do!